build-cdi-func-test-file-host.sh causes permission denied when being build

How to reproduce:
* clone repo under regular user (non-root)
*  make functests

```
[lbednar@lbednar tmp]$ git clone https://github.com/kubevirt/containerized-data-importer.git
Cloning into 'containerized-data-importer'...
remote: Counting objects: 12160, done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 12160 (delta 1), reused 1 (delta 0), pack-reused 12153
Receiving objects: 100% (12160/12160), 46.24 MiB | 1.80 MiB/s, done.
Resolving deltas: 100% (4851/4851), done.
[lbednar@lbednar tmp]$ cd containerized-data-importer/
[lbednar@lbednar containerized-data-importer]$ make functest
./hack/build/in-docker.sh "DOCKER_REPO= DOCKER_TAG= VERBOSITY= PULL_POLICY= ./hack/build/build-manifests.sh"
Sending build context to Docker daemon 4.608 kB
Step 1/7 : FROM fedora:27
 ---> 9110ae7f579f
Step 2/7 : RUN dnf install -y qemu xz gzip git && dnf clean all
 ---> Using cache
 ---> 008c451e628d
Step 3/7 : ENV GIMME_GO_VERSION 1.10 GOPATH "/go" GOBIN "/usr/bin"
 ---> Using cache
 ---> 02b8eedd54bf
Step 4/7 : RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=/gimme bash >> /etc/profile.d/gimme.sh
 ---> Using cache
 ---> bbc040a5b2cb
Step 5/7 : RUN mkdir -p ${GOPATH} &&     source /etc/profile.d/gimme.sh &&     eval $(go env) &&     (go get -u github.com/onsi/ginkgo/ginkgo &&      cd $GOPATH/src/github.com/onsi/ginkgo/ginkgo &&      go install ./... ) &&     go get golang.org/x/tools/cmd/goimports &&     ( go get -d mvdan.cc/sh/cmd/shfmt || echo "**** Expecting error \"cannot find package mvdan.cc/sh/v2/fileutil\"" ) &&     ( cd $GOPATH/src/golang.org/x/tools/cmd/goimports &&     go install ./... ) &&     ( cd $GOPATH/src/mvdan.cc/sh/cmd/shfmt &&     git checkout v2.5.0 -b build-v2.5.0 &&     go install ./... ) &&     ( go get -d github.com/mattn/goveralls &&     cd $GOPATH/src/github.com/mattn/goveralls &&     go install ./... )
 ---> Running in df4a29665649
go version go1.10 linux/amd64
Switched to a new branch 'build-v2.5.0'
 ---> e4d7ae7b5bbd
Removing intermediate container df4a29665649
Step 6/7 : ADD entrypoint.sh /entrypoint.sh
 ---> 891b8c3a85ff
Removing intermediate container 60abe2733ad5
Step 7/7 : ENTRYPOINT /entrypoint.sh
 ---> Running in e2385b4abee8
 ---> 4c1f75b82d5f
Removing intermediate container e2385b4abee8
Successfully built 4c1f75b82d5f
go version go1.10 linux/amd64
./hack/build/in-docker.sh "./hack/build/build-go.sh clean && ./hack/build/build-go.sh build tools/cdi-func-test-file-host-init && DOCKER_REPO= DOCKER_TAG= VERBOSITY= PULL_POLICY= ./hack/build/build-manifests.sh tools/cdi-func-test-file-host-init && ./hack/build/build-copy-artifacts.sh tools/cdi-func-test-file-host-init"
Sending build context to Docker daemon 4.608 kB
Step 1/7 : FROM fedora:27
 ---> 9110ae7f579f
Step 2/7 : RUN dnf install -y qemu xz gzip git && dnf clean all
 ---> Using cache
 ---> 008c451e628d
Step 3/7 : ENV GIMME_GO_VERSION 1.10 GOPATH "/go" GOBIN "/usr/bin"
 ---> Using cache
 ---> 02b8eedd54bf
Step 4/7 : RUN mkdir -p /gimme && curl -sL https://raw.githubusercontent.com/travis-ci/gimme/master/gimme | HOME=/gimme bash >> /etc/profile.d/gimme.sh
 ---> Using cache
 ---> bbc040a5b2cb
Step 5/7 : RUN mkdir -p ${GOPATH} &&     source /etc/profile.d/gimme.sh &&     eval $(go env) &&     (go get -u github.com/onsi/ginkgo/ginkgo &&      cd $GOPATH/src/github.com/onsi/ginkgo/ginkgo &&      go install ./... ) &&     go get golang.org/x/tools/cmd/goimports &&     ( go get -d mvdan.cc/sh/cmd/shfmt || echo "**** Expecting error \"cannot find package mvdan.cc/sh/v2/fileutil\"" ) &&     ( cd $GOPATH/src/golang.org/x/tools/cmd/goimports &&     go install ./... ) &&     ( cd $GOPATH/src/mvdan.cc/sh/cmd/shfmt &&     git checkout v2.5.0 -b build-v2.5.0 &&     go install ./... ) &&     ( go get -d github.com/mattn/goveralls &&     cd $GOPATH/src/github.com/mattn/goveralls &&     go install ./... )
 ---> Using cache
 ---> e4d7ae7b5bbd
Step 6/7 : ADD entrypoint.sh /entrypoint.sh
 ---> Using cache
 ---> 891b8c3a85ff
Step 7/7 : ENTRYPOINT /entrypoint.sh
 ---> Using cache
 ---> 4c1f75b82d5f
Successfully built 4c1f75b82d5f
go version go1.10 linux/amd64
/go/src/kubevirt.io/containerized-data-importer/_out/tools/cdi-func-test-file-host-init/
./hack/build/build-cdi-func-test-file-host.sh
mkdir: cannot create directory ‘/tmp/containerized-data-importer/_out/tools/cdi-func-test-file-host-http’: Permission denied
make: *** [Makefile:115: .functest-image-host] Error 1
```
That target was introduced in #315

--- a/Makefile
+++ b/Makefile
@@ -107,9 +107,9 @@ cluster-sync-importer: cluster-sync
 cluster-sync-cloner: WHAT = cmd/cdi-cloner
 cluster-sync-cloner: cluster-sync
 
-functest: .functest-image-host
+functest:
 	./hack/build/functests.sh
 
-.functest-image-host: WHAT=tools/cdi-func-test-file-host-init
-.functest-image-host:  manifests build
-	./hack/build/build-cdi-func-test-file-host.sh
+functest-image-host: WHAT=tools/cdi-func-test-file-host-init
+functest-image-host:  manifests build
+	${DO} ./hack/build/build-cdi-func-test-file-host.sh && ./hack/build/build-docker.sh "tools/cdi-func-test-file-host-init tools/cdi-func-test-file-host-http"

--- a/hack/build/build-cdi-func-test-file-host.sh
+++ b/hack/build/build-cdi-func-test-file-host.sh
@@ -34,7 +34,3 @@ ${BUILD_DIR}/build-copy-artifacts.sh "${FILE_INIT_PATH}"
 OUT_PATH="${OUT_DIR}/tools"
 cp ${BUILD_DIR}/docker/${FILE_HOST}/* ${OUT_PATH}/${FILE_HOST}/
 cp "${CDI_DIR}/test/images/tinyCore.iso" "${OUT_PATH}/${FILE_INIT}/"
-
-for target in "${FILE_HOST}" "${FILE_INIT}"; do
-	${BUILD_DIR}/build-docker.sh build "tools/${target}"
-done

